### PR TITLE
Try to add startup commands

### PR DESCRIPTION
**Pull request by hmaarrfk**
<details>
The general idea is to use the EEPROM.

We check for exactness in the version and if the version matches we then run the startup commands as if they are coming from the serial port.

I'm not so much of a fan of using the EEPROM, i wish i could use the program memory.

</details>

New idea:

1. Just define a list at compile time of what you want.
2. Try to make it easy for users to override it (trying to use `__attribute__((weak))`)
3. ~Let users use this as a library.~

The main purpose of this is to enable the setup of "safe" pins and "startup routines" that might not need error checking. If branching is required then this cannot be used as is.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
https://github.com/PaulStoffregen/LittleFS

it is somewhat ideal since it gets wiped out??

**Comment by hmaarrfk** [comment]
hmm, this is somewhat of limited use since there is little feedback that can be created without an overseeing script....

**Comment by hmaarrfk** [comment]
I guess its kinda OK since we can use this to test EXTERNAL hardware.

File: src/commandconstants.hpp [hunk 1]
```diff
@@ -60,12 +60,19 @@ int register_write_uint16(CommandRouter *cmd, int argc, const char **argv);
...
+
+// Mostly for debugging and startup scripts
+int sleep_seconds(CommandRouter *cmd, int argc, const char **argv);
+int startup_commands_available(CommandRouter *cmd, int argc, const char **argv);
+int read_startup_command(CommandRouter *cmd, int argc, const char **argv);
+int demo_commands_available(CommandRouter *cmd, int argc, const char **argv);
+int read_demo_command(CommandRouter *cmd, int argc, const char **argv);
+int disable_demo_commands(CommandRouter *cmd, int argc, const char **argv);
+int enable_demo_commands(CommandRouter *cmd, int argc, const char **argv);
+int demo_commands_available(CommandRouter *cmd, int argc, const char **argv);
```
Comment (vetonsaliu8) [inline src/commandconstants.hpp:74]: @hmaarrfk is this a duplicate?

File: src/commandrouting.cpp [hunk 2]
```diff
@@ -230,31 +257,7 @@ int CommandRouter::processSerialStream() {
...
-    argv[argc] = &input_buffer[i];
-    argc++;
-    while(i < bytes_read) {
-        i += 1;
-        c = input_buffer[i];
-        if (c == ' ' || c  == '\0')
-            break;
-    }
-  }
+  parseInputBuffer(input_buffer, bytes_read, argv, argc);
```
Comment (vetonsaliu8) [inline src/commandrouting.cpp:260]: is this right? argv is not declared anywhere in CommandRouter::processSerialStream

File: src/commandconstants.hpp (hunk 1 above)
Comment (hmaarrfk) [reply src/commandconstants.hpp:74]: Yes. It seems so

File: src/commandrouting.cpp (hunk 2 above)
Comment (hmaarrfk) [reply src/commandrouting.cpp:260]: They should be part of the class. I think

**Pull request merged**